test/mlkem_internal_test.c

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2024 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <openssl/opensslconf.h>
+#ifndef OPENSSL_NO_STDIO
+# include <stdio.h>
+#endif
+
+#ifndef OPENSSL_NO_MLKEM
+# include <crypto/mlkem.h>
+#endif
+
+#include <string.h>
+#include "testutil.h"
+#include "testutil/output.h"
+
+int main(void)
```

Comment:
this should be implemented as a standard test, i.e. using the test framework with a `setup_tests()` function etc

Reply:
AFAIK @andrewkdinh already has that in the works, so leaving as-is for now.